test/README.md

```diff
@@ -92,3 +108,59 @@ When a new version of Docker is released, you willneed to update the GCE test im
 To do this, change the Docker version in `run-integration-tests.sh` and push the change.
 Next build in CircleCI will detect that there is no template for this version of Docker and will first create the template before running tests.
 Subsequent builds will then simply re-use the template.
+
+## Running the tests on a cloud provider
+
+Running the tests against a remote provider instead of local vagrant VMs is a little more involved, and reproduces the environment used for CI. It is recommended that you use the build VM to coordinate the build phase.
```

Comment:
> It is recommended that you use the build VM to coordinate the build phase.

Why so? (You may want to document the reasons?)
I used to create a build VM in GCP, and then to copy the docker images across to some other test VMs in GCP.

Reply:
Just less hassle really, it's a pre-configured environment that is known to work reproducibly. This isn't the case when building locally, and using a cloud VM is more hassle (by a few commands) than a local one.